src/System.Runtime.Intrinsics.X86/src/System.Runtime.Intrinsics.X86.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <AssemblyName>System.Runtime.Intrinsics.X86</AssemblyName>
+    <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp-Debug|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp-Release|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp-Windows_NT-Debug|AnyCPU'" />
```

Comment:
Only keep the first two propertygroups for configurations, remove the other 4.

Reply:
I tried, but an error occurred: 
```
D:\workspace\corefx\buildvertical.targets(162,5): error : Could not find a configuration for ProjectReference 'D:\workspace\corefx\\external\runtime\runti
me.depproj' from configurations 'netcoreapp-Windows_NT;netcoreapp-Unix;uap;uapaot;mono' when building 'System.Runtime.Intrinsics.X86' for configuration 'n
etcoreapp-Release' . [D:\workspace\corefx\src\System.Runtime.Intrinsics.X86\src\System.Runtime.Intrinsics.X86.csproj] Command execution failed with exit code 1.
```

Reply:
Hmm, are you sure that was related to removing these <PropertyGroup>'s? These actually shouldn't have any effect on the command-line build -- they are only here to tell Visual Studio what configurations are available. The command-line build should be influenced by what is in Configurations.props.